src/EnergyPlus/General.cc

```diff
@@ -157,7 +152,7 @@ constexpr std::array<std::string_view, static_cast<int>(RptKey::Num)> RptKeyName
     "COSTINFO", "DXF", "DXF:WIREFRAME", "VRML", "VERTICES", "DETAILS", "DETAILSWITHVERTICES", "LINES"};
 
 // A second version that does not require a payload -- use lambdas
-void SolveRoot(EnergyPlusData &state,
+void SolveRoot(const EnergyPlusData &state,
```

Comment:
[src/EnergyPlus/General.cc:160]:(style),[constParameter],Parameter 'state' can be declared as reference to const

Reply:
Noting the const-ness of state will be very helpful in the future if we decide to be more particular about how much of state we are passing around.  It's helpful now either way.